numpy-stubs/__init__.pyi

```diff
@@ -792,3 +797,305 @@ class AxisError(ValueError, IndexError):
     def __init__(
         self, axis: int, ndim: Optional[int] = ..., msg_prefix: Optional[str] = ...
     ) -> None: ...
+
+# Functions from np.core.fromnumeric
+_Mode = Literal["raise", "wrap", "clip"]
+_Order = Literal["C", "F", "A"]
+_PartitionKind = Literal["introselect"]
+_SortKind = Literal["quicksort", "mergesort", "heapsort", "stable"]
+_Side = Literal["left", "right"]
+
+# A placeholder (?) for numerical scalars
+_Scalar = TypeVar("_Scalar", bound=Union[int, float, complex, bool, bool_, number])
```

Comment:
Once ``np.ndarray`` is a proper Generic I'd imagine it will be easier to define ``_Scalar ``, ``_ArrayLikeInt`` and ``_ArrayLikeBool`` here.